src/web-ui/src/app/app-routing.module.ts

```diff
@@ -112,7 +112,9 @@ const ROUTES: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(ROUTES)],
+  imports: [RouterModule.forRoot(ROUTES, {
+    scrollPositionRestoration: 'enabled'
```

Comment:
Do czego to jest?

Reply:
Czego nie rozumiesz xd jak jesteś zescrollowany gdzieś w połowie czy na dole strony i klikniesz w link do routingu domyślnie nie ustawi Cię na górze strony, dzięki tej opcji podczas przejścia do innego komponentu focusuje Cię na górze strony

Reply:
No właśnie chciałem wyjaśnienia od czego jest property `scrollPositionRestoration` bo nigdy go nie widziałem.